iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/ShortBackupRetentionCheck.java

```diff
@@ -37,6 +37,11 @@
     defaultValue = "" + DEFAULT)
   int backupRetentionDuration = DEFAULT;
 
+  @Override
+  protected void registerChecks() {
+    // do not register any check for a specific resource type
```

Comment:
If this is so common, why not having a default empty implementation for `registerChecks` in `AbstractResourceCheck`?